vipps-recurring-api.md

```diff
@@ -621,13 +623,36 @@ to set up a new agreement.
 | 1 | `PENDING`  | The charge has been created, but has not yet been approved by the user. |
 | 2 | `DUE`      | The charge is now visible in Vipps, and will be made in the next 30 days (this was previously 6). |
 | 3 | `CHARGED`  | The charge has been completed. |
-| 4 | `FAILED`   | The charge has failed for some reason, i.e. expired card, insufficient funds, etc. |
+| 4 | `FAILED`   | The charge has failed for some reason, i.e. expired card, insufficient funds, etc. Read the [Charge failure reasons](https://github.com/vippsas/vipps-recurring-api/blob/master/vipps-recurring-api.md#charge-failure-reasons) section for more details. |
 | 5 | `REFUNDED` | The charge has been refunded. The timeframe for issuing a refund is 365 days from the date of capture. |
 | 6 | `PARTIALLY_REFUNDED`| A part of the captured amount has been refunded. |
 | 7 | `RESERVED` | An initial charge with `transactionType` set to `RESERVE_CAPTURE` changes state to `CHARGED` when captured successfully. |
 | 8 | `CANCELLED` | The charge has been cancelled. |
 | 9 | `PROCESSING` | The charge is currently being processed by Vipps. |
 
+### Charge failure reasons
+
+When fetching a charge through the API, you can find two fields in the response body to identify why the charge failed `failureReason` and `failureDescription`:
+
+```json
+{
+  "status": "FAILED",
+  "type": "RECURRING",
+  "failureReason": "insufficient_funds",
+  "failureDescription": "Payment was declined by the payer bank due to lack of funds"
+}
+```
+
+Here is a list of possible values for `failureReason`, their respective descriptions and possible actions that the user/merchant could take.
+
+| Reason | Description | Action |
+| ---- | ----------- | ------ |
+| insufficient_funds | Payment was declined by the payer bank due to lack of funds. | User should fund the card to cover the difference between the amount to be paid or use another payment source that is adequately funded to complete the transaction. |
+| invalid_card | The user tried to pay using a card that has either expired or is disabled by the issuer. | User must change payment source |
+| verification_required | Payment declined because the issuing bank requires verification. | |
```

Comment:
```suggestion
| verification_required | Payment declined because the issuing bank requires verification. | Ask the user to change, or add a new, payment source on their agreement in Vipps. Alternatively removing and then adding the card might solve the issue. |
```